Packages/doc/releasenotes_new.rst

```diff
@@ -0,0 +1,244 @@
+44253e11 (Merge pull request #924 from AllenInstitute/feature/924-add-branch-naming-scheme-documentation, 2021-04-23)
+
+Release 2.2
+===========
+
+This version is the first version of MIES with support for upcoming Igor Pro 9. And it will also be the last with
+support for Igor Pro 8.
+
+Sweep Formula
+-------------
+
+- Add ``area`` operation
+- Fix most cases of ignored minus signs in formulas
+- Make it more user friendly on errors by not just asserting out
+
+
+AnalysisBrowser
+---------------
+
+
+DataBrowser
+-----------
+
+Dashboard
+~~~~~~~~~
+
+- Display the headstages for each sweep
+- Add support for multipatch seq analysis functions
+
+
+DataBrowser/SweepBrowser
+------------------------
+
+- Fix restoring from backup so that it works again and also restores all sweeps and not only the displayed ones
+- The axes' locations are now more predictable for zomming in/out via mouse wheel
+- Don't display data from TP during DAQ channels
+- Always create backup waves when splitting the sweep
+- Speedup plotting of many traces by grouping AppendToGraph calls. This is mostly noticable in Igor Pro 8, les in
+  Igor Pro 9.
+- Make overlay sweeps with headstage removal faster
+- Select the current sweep when enabling overlay sweeps
+- Add new trace popup for convenient stimulus set opening in the wavebuilder
+- Fix displaying TTL data with only "TP during DAQ" data
+
+
+DA\_Ephys
+---------
+
+- Closing the DAEphys panel that is running TP now waits for the TP analysis to finish before closing
+- Ignore TP during DAQ channels for locked indexing
+- Make sweep skipping honour the set borders as otherwise break indexing
+- Handle relocking a device with data better. We now don't start at sweep zero but at next sweep after the last acquired
+- Remove decimation factor control
+- Make rerunning the TP with the same settings faster
+- Don't disable dDAQ and oodDAQ checkboxes during data aquisition anymore
+- Don't allow dialogs during background function execution, this should prevent Igor Pro crashes
+- Add new trace popup for convenient stimulus set opening
```

Comment:
@t-b Alternate method for opening a stimulus set in the wavebuilder; right click trace in data browser, select `open stimulus set in wavebuilder'

but in analysis browser I get:
```
  !!! Assertion FAILED !!!
  Message: "str: "Y1_SPulsTrn_20Hz_DA_0" is not in the popupmenus "popup_WaveBuilder_SetList" list"
  Please provide the following information if you contact the MIES developers:
  ################################
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  Stacktrace:
WB_OpenStimulusSetInWaveBuilder(...)#L68 [MIES_WaveBuilderPanel.ipf]
PGC_SetAndActivateControl(...)#L290 [MIES_ProgrammaticGUIControl.ipf]
SetPopupMenuString(...)#L487 [MIES_GuiUtilities.ipf]
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  Time: 2021-04-30T14:01:04-07:00
  Locked device: [- none -]
  Current sweep: [- none -]
  DAQ: [- none -]
  Testpulse: [- none -]
  Experiment: Untitled ()
  Igor Pro version: 9.0.0.5 (37373)
  MIES version:
  Release_2.1_20200723-1170-gff6f51a6
Date and time of last commit: 2021-04-30T19:45:05+02:00
Submodule status: 
160000 3ca8f77c798c04e432907c95e7a177bc2905bec3 0	Packages/IPNWB
160000 5ffbdab2c5d42b54f86054c9bcd0cea6e5f07829 0	Packages/doc/doxygen-filter-ipf
160000 42276ed677273c02bc7b457324081dd463bb0d6b 0	Packages/unit-testing
  ################################
```

Reply:
I've added an issue to fix that.